src/Calculator/Views/GraphingCalculator/EquationInputArea.xaml

```diff
@@ -2,67 +2,269 @@
+                                             Style="{StaticResource ThemedVariableTextBoxStyle}"
+                                             GotFocus="TextBoxGotFocus"
+                                             KeyDown="TextBoxKeyDown"
+                                             LosingFocus="TextBoxLosingFocus"
+                                             Text="{x:Bind Min, Mode=OneWay}"/>
+                                </StackPanel>
+                                <StackPanel Grid.Column="1"
+                                            HorizontalAlignment="Center"
+                                            Orientation="Horizontal">
+                                    <TextBlock x:Uid="StepTextBlock"
+                                               Margin="0,4,0,0"
+                                               HorizontalAlignment="Center"
+                                               VerticalAlignment="Center"/>
+                                    <TextBox x:Name="StepTextBox"
+                                             HorizontalAlignment="Center"
+                                             Style="{StaticResource ThemedVariableTextBoxStyle}"
+                                             GotFocus="TextBoxGotFocus"
+                                             KeyDown="TextBoxKeyDown"
+                                             LosingFocus="TextBoxLosingFocus"
+                                             Text="{x:Bind Step, Mode=OneWay}"/>
+                                </StackPanel>
+                                <StackPanel Grid.Column="2" Orientation="Horizontal">
+                                    <TextBlock x:Uid="MaxTextBlock"
+                                               Margin="0,4,0,0"
+                                               VerticalAlignment="Center"/>
+                                    <TextBox x:Name="MaxTextBox"
+                                             HorizontalAlignment="Left"
+                                             Style="{StaticResource ThemedVariableTextBoxStyle}"
+                                             GotFocus="TextBoxGotFocus"
+                                             KeyDown="TextBoxKeyDown"
+                                             LosingFocus="TextBoxLosingFocus"
+                                             Text="{x:Bind Max, Mode=OneWay}"/>
+                                </StackPanel>
+                            </Grid>
+                        </StackPanel>
+                    </Border>
+                </Grid>
+            </DataTemplate>
+
+        </ResourceDictionary>
+    </UserControl.Resources>
+    <Grid VerticalAlignment="Top">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="*"/>
+            <RowDefinition Height="Auto"/>
+        </Grid.RowDefinitions>
+        <mux:ItemsRepeaterScrollHost>
+            <ScrollViewer>
+                <StackPanel>
+                    <mux:ItemsRepeater x:Name="EquationInputList"
```

Comment:
nit: When I hover on the right side of any equation an empty scroll bar appears even though there is nothing to scroll